src/thrift-netcore/thrift-netcore.csproj

```diff
@@ -49,4 +53,8 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Net.NameResolution" Version="4.3.0" />
+  </ItemGroup>
```

Comment:
Could you maybe group it with the others line 25? I think it should only be required for NetStd 1.x